redhat-developer/developers.redhat.com
# DO NOT MERGE [DEVELOPER-5529] Connectors no DCP

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5574
* https://issues.jboss.org/browse/DEVELOPER-5529

### Verification Process

* Go to the connectors page (/products/fuse/connectors)
* Verify no calls are made to the DCP occur on load
* Verify no calls are made to the DCP on sort and ordering
* Verify sorting is working as expected
* Verify images display for each connectors
